plugins/services/src/js/utils/ServiceActionModalsUtil.ts

```diff
@@ -0,0 +1,6 @@
+export function getActionModalReadableError(errorMsg: any): any {
```

Comment:
You can import `i18n` from `container` and update this to use it inside the `if` to return the translated text and otherwise default to the untranslated errorMsg.